climate_emotions_map/make_descriptive_plots.py

```diff
@@ -209,85 +266,164 @@ def make_impact_plot_traces(df: pd.DataFrame):
-            [{"colspan": 3}, None, None, {"colspan": 3}, None, None],
-            [{"colspan": 6}, None, None, None, None, None],
-            [{"colspan": 6}, None, None, None, None, None],
-        ],
+        rows=len(row_heights),
+        cols=1,
+        row_heights=row_heights,
         subplot_titles=[
             get_demographic_variable_to_display(demographic_variable)
-            for demographic_variable in SUBPLOT_POSITIONS.keys()
+            for demographic_variable in SUBPLOT_POSITIONS
         ],
+        vertical_spacing=0.04,
+    )
+
+    colorscale_step = 1 / (len(row_heights) - 1)
+    colors = sample_colorscale(
+        "turbo", np.arange(0, 1 + colorscale_step, colorscale_step)
     )
 
     # add plots
+    demographic_variable_labels = []
     for demographic_variable in SUBPLOT_POSITIONS.keys():
 
         row, col = SUBPLOT_POSITIONS[demographic_variable]
+        color = colors[row - 1]
 
         if demographic_variable == IMPACTS_LABEL:
-            for trace in make_impact_plot_traces(data):
-                fig.add_trace(trace, row=row, col=col)
+            traces = make_impact_plot_traces(
+                data, text_wrap_width=text_wrap_width, marker_color=color
+            )
+        else:
+            traces = make_descriptive_plot_traces(
+                data, demographic_variable, reverse=True, marker_color=color
+            )
+            if demographic_variable != Q2_LABEL:
+                demographic_variable_labels.extend(
+                    [demographic_variable] * len(traces[0].y)
+                )
+
+        for trace in traces:
+            fig.add_trace(trace, row=row, col=col)
+
+        if not demographic_variable == IMPACTS_LABEL:
+            tickvals = ["" for _ in CATEGORY_ORDERS[demographic_variable]]
+            ticktext = tickvals
+
+            fig.update_yaxes(
+                tickvals=tickvals,
+                ticktext=ticktext,
+                row=row,
+                col=col,
+            )
+            fig.update_xaxes(
+                range=[0, 105],
+                tickvals=[0, 20, 40, 60, 80, 100],
+                ticktext=["0", "20", "40", "60", "80", "100 (%)"],
+                row=row,
+                col=col,
+            )
+            fig.update_layout(bargap=0)
+
         else:
-            fig.add_trace(
-                make_descriptive_plot(data, demographic_variable),
+            fig.update_yaxes(
+                range=[0, 100],
+                tickvals=[0, 20, 40, 60, 80, 100],
+                ticktext=["0", "20", "40", "60", "80", "(%)"],
+                row=row,
+                col=col,
+            )
+            fig.update_xaxes(
+                tickvals=[],
+                ticktext=[],
                 row=row,
                 col=col,
             )
 
-    # turn off legend
-    fig.update_layout(showlegend=False)
+        fig.update_yaxes(
+            tickfont=dict(size=10),
```

Comment:
no benefit to calling the `dict` constructor here, I think
```suggestion
            tickfont={"size": 10},
```